Porting across bloom documentation (#2375)
ros2/ros2_documentation: source/How-To-Guides/Releasing/Release-Team-Repository.rst

Release Team / Repository
=========================

.. contents:: Table of Contents

>    :depth: 3

Reviewer: ```suggestion
   :depth: 2
```

   :local:

This page explains the recommended method of hosting your release repositories on `ros2-gbp <https://github.com/ros2-gbp>`_.

What is ROS 2 GBP?
------------------

`ros2-gbp <https://github.com/ros2-gbp>`_ is a GitHub organization that hosts the release repositories for ROS packages.

> They also maintain a list of release teams, the list of members per release team and the list of release repositories maintained by the release teams.

Reviewer: ```suggestion
It also maintains a list of release teams, the list of members per release team and the list of release repositories maintained by the release teams in https://github.com/ros2-gbp/ros2-gbp-github-org.
```

> Interactions with ros2-gbp are done through raising GitHub issues.

Reviewer: ```suggestion
Interactions with ros2-gbp-github-org are done through raising GitHub issues.
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Fill the `Add New Release Repositories issue <https://github.com/ros2-gbp/ros2-gbp-github-org/issues/new?assignees=&labels=&template=new_release_repository.md&title=Add+new+release+repositories>`_ issue template
if you don't have a release repo for your project yet.

What if my existing release repo isn't on ros2-gbp?
---------------------------------------------------

Packages released before ros2-gbp existed may have their release repositories hosted elsewhere.
It is now strongly recommended for release repositories to live in this dedicated GitHub organization.
If you are porting a ROS 1 package to ROS 2 and planning on releasing your packages into ROS 2 for the first time, follow standard procedure to request for a new release repository for your ROS 2 releases.
If you have previously released your packages for ROS 2, when raising the `Add New Release Repositories issue <https://github.com/ros2-gbp/ros2-gbp-github-org/issues/new?assignees=&labels=&template=new_release_repository.md&title=Add+new+release+repositories>`_, **specify your current release repository url**, and follow standard procedure for the rest.

.. note::

>    Release repositories hosted elsewhere are still supported for stable distributions if you are not planning to release the repository into {DISTRO}.

Reviewer: ```suggestion
   Release repositories hosted elsewhere are still supported for stable distributions if you are not planning to release the repository into Rolling.
```

(I think we want this to always say "Rolling", even on other branches)
